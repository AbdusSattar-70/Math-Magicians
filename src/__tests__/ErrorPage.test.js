import renderer from 'react-test-renderer';
import Errorpage from '../pages/ErrorPage';

it('renders correctly', () => {
  const tree = renderer
    .create(<Errorpage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
