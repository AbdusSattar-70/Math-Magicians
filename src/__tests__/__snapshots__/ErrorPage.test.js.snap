// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Error: 404, Page Not Found
  </h1>
</div>
`;
